src/styles/blocks/_card.scss

```
@@ -0,0 +1,82 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin-bottom: 46px;
+
+  &__picture {
+    width: 160px;
+    height: 134px;
+    align-self: center;
+  }
+
+  &__product-name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $font-main-color;
+    padding-top: 40px;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    color: $font-secondary-color;
+    padding-top: 4px;
+  }
+
+  &__rating-panel {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-top: 16px;
+  }
+
+  &__review-counter {
+    font-size: 10px;
+    color: $font-main-color;
+  }
+
+  &__price-block {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 24px 0 16px;
+  }
+
+  &__price-label {
+    font-size: 12px;
+    color: $font-secondary-color;
+  }
+
+  &__price {
+    font-size: 16px;
+    font-weight: 700;
+    color: $font-main-color;
+  }
+
+  &__buy-button {
+    display: flex;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    color: #fff;
```

You create variable for colors, but not for all 🥺 
```suggestion
    color: $c-white;
```

---

Done